src/style.scss

```diff
@@ -297,6 +297,30 @@ $maxCaptionHeight: 150px;
   right: 0;
   bottom: 0;
   left: 0;
+  .imagePrev & {
+    display: none;
```

Comment:
Does this not cause issues with animation when the adjacent images load faster?

Reply:
I do this so not to display the loading and/or error message on the previous or next images on the current image

Reply:
Aaah, so that's the source of that bug. Thanks!